features/docs/gegeven-stap-definities-expressief.feature

```diff
@@ -0,0 +1,155 @@
+  #language: nl
+
+  @stap-documentatie
+  Functionaliteit: Persoon, Inschrijving, Verblijfplaats gegeven stap definities
+
+  Achtergrond:
+  Gegeven de 1e 'SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl' statement heeft als resultaat '9999'
+  En de 2e 'SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl' statement heeft als resultaat '10000'
+  En de 3e 'SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl' statement heeft als resultaat '10001'
+
+  Scenario: de persoon met burgerservicenummer '[bsn]'
+      Gegeven de persoon met burgerservicenummer '000000012'
+      Dan zijn de gegenereerde SQL statements
+      | stap | categorie    | text                                                                                                                                                  | values               |
+      | 1    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING * | 0                    |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr) VALUES($1,$2,$3,$4,$5)                                      | 9999,0,0,P,000000012 |
+
+  Scenario: is minderjarig
+    Gegeven de persoon met burgerservicenummer '000000012'
+    * is minderjarig
+      Dan zijn de gegenereerde SQL statements
+      | stap | categorie    | text                                                                                                                                                  | values                                  |
+      | 1    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING * | 0                                       |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geboorte_datum) VALUES($1,$2,$3,$4,$5,$6)                    | 9999,0,0,P,000000012,gisteren - 17 jaar |
+
+  Scenario: de persoon '[aanduiding]' met burgerservicenummer '[bsn]' heeft inschrijving
```

Comment:
Is het een idee om als titel 'is ingeschreven in de BRP' te gebruiken?
Deze stap definitie werkt ook met andere 'persoon' stap definities zoals de ene waar geen aanduiding is opgegeven